mutmut/__init__.py

```diff
@@ -1028,7 +1028,7 @@ def kill(process_):
                     if not line:
                         break
                     callback(line)
-        except (IOError, OSError):
+        except OSError:
```

Comment:
See https://docs.python.org/3/library/exceptions.html#OSError

> Changed in version 3.3: EnvironmentError, IOError, WindowsError, socket.error, select.error and mmap.error have been merged into OSError, and the constructor may return a subclass.

Reply:
Nice. I will try to remember that.